docs/release-notes/_category_.json

```diff
@@ -0,0 +1,3 @@
+{
+  "label": "Release notes"
```

Comment:
@jooola Instead of doing it this way, I'd recommend we put the release notes in a [separate docs instance](https://docusaurus.io/docs/docs-multi-instance) so we can keep the sections and their search indexes separate.

Reply:
Sorry I am unsure about what you want to accomplish. Could you explain the end goal ? From what I understand we are not heading toward a versioned documentation, so having multiple docs instance does not seems necessary based on the docusaurus docs.